modules/ROOT/pages/dataweave-formats-java.adoc

```diff
@@ -41,19 +41,13 @@ The following table maps Java classes to DataWeave types.
 | `java.util.Iterator` | `Array`
 | `java.util.Map` | `Object`
 | `java.util.Optional` | `Null` or `Value`
```

Comment:
All these need to be restored, probably added to source.
They were added to v2.4 via https://github.com/mulesoft/docs-dataweave/pull/205/files

Reply:
Updating data-weave master branch via https://github.com/mulesoft/data-weave/pull/2042/files
Also need to update support/2.5.0

Reply:
fixed via PR above